projects/ddap-common-lib/src/lib/layout/menu/menu.component.html

```diff
@@ -4,13 +4,26 @@
 -->
 <mat-toolbar color="primary" class="main-header">
   <div class="menus">
-    <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
-    <span *ngIf="isSandbox"
-      class="sandbox-warning"
-      matTooltip="This is sandbox testing environment">
-      Sandbox environment
-    </span>
-    <a class="site-name" routerLink="/">{{siteName}}</a>
+    <ng-container *ngIf="logoLink; else noLogoLink">
+      <a href="{{logoLink}}">
+        <img *ngIf="logoUrl" class="m-r" style="height: 32px; width: auto;" src="{{logoUrl}}"/>
+      </a>
+      <span *ngIf="isSandbox"
+        class="sandbox-warning"
+        matTooltip="This is sandbox testing environment">
+        Sandbox environment
+      </span>
```

Comment:
I don't like the code duplicate here. I think this block should come before the logo or after the site name so that we don't need an extra copy? What do you think, @monicavalluri, @mpanik?